src/coreclr/jit/target.h

```diff
@@ -371,6 +356,18 @@ static bool operator>(regMaskTP first, regMaskTP second)
     return first.getLow() > second.getLow();
 }
 
+static regMaskTP operator<<(regMaskTP& first, const int b)
+{
+    regMaskTP result(first.getLow() << b);
+    return result;
+}
+
+static regMaskTP operator>>(regMaskTP& first, const int b)
```

Comment:
These shouldn't be taking `first` by (mutable) reference since they aren't the compound assignment versions. I'd suggest we just make it by value.

Reply:
Yes, I changed bunch of these but forgot to change for `<<` and `>>`. Will fix in next PR.